src/test/java/teammates/test/cases/ui/InstructorFeedbackAddActionTest.java

```diff
@@ -86,8 +89,9 @@ public void testExecuteAndPostProcess() {
         
         ______TS("Error: Invalid parameter (invalid sesssion name, > 38 characters)");
         
+        String longFsName = "123456789012345678901234567890123456789";
```

Comment:
I believe we have a utility method that can generate a string of exact length. We should use that to generate a string of exactly 39 letters.